app/bundles/IntegrationsBundle/EventListener/CompanyObjectSubscriber.php

```diff
@@ -81,7 +81,6 @@ public function updateCompanies(InternalObjectUpdateEvent $event): void
                 $event->getUpdateObjects()
             )
         );
-        $event->stopPropagation();
```

Comment:
@kuzmany could you explain why we need to remove this line? I couldn't find it in the PR description. Thanks!

Reply:
I describe it in description. We made integration what need process objects after processed it by IntegrationsBundle. 
We need created/updated object Ids post to another API. 

```php
class ContactObjectSubscriber implements EventSubscriberInterface
{

    /**
     * @return array
     */
    public static function getSubscribedEvents()
    {
        return [
            IntegrationEvents::INTEGRATION_UPDATE_INTERNAL_OBJECTS => ['updateContacts', -1],
            IntegrationEvents::INTEGRATION_CREATE_INTERNAL_OBJECTS => ['createContacts', -1],
        ];
    }

    public function updateContacts(InternalObjectUpdateEvent $event): void
    {
        if (Contact::NAME !== $event->getObject()->getName()) {
            return;
        }

        // do whatever you want with $event->getUpdatedObjectMappings()
    }

    public function createContacts(InternalObjectCreateEvent $event): void
    {
        if (Contact::NAME !== $event->getObject()->getName()) {
            return;
        }
        // work whatever you want with $event->getObjectMappings()
    }
}
```

Reply:
Ah okay. From https://symfony.com/doc/current/components/event_dispatcher.html#stopping-event-flow-propagation: 

> In some cases, it may make sense for a listener to prevent any other listeners from being called. In other words, the listener needs to be able to tell the dispatcher to stop all propagation of the event to future listeners (i.e. to not notify any more listeners). This can be accomplished from inside a listener via the `stopPropagation()` method.

Makes sense now 😊

Reply:
This should be fine. I see no other reason for stopping the propagation other than a small performance gain that it does not have to check the other object. It should not break any funciton.